wasm/javascript-v8/instance.go

```diff
@@ -0,0 +1,28 @@
+package v8
+
+import (
+	"context"
+
+	"rogchap.com/v8go"
+)
+
+type V8Instance struct {
+	iso *v8go.Isolate
+	ctx *v8go.Context
+}
+
+func NewV8Instance(iso *v8go.Isolate) (*V8Instance, error) {
+	ctx := v8go.NewContext(iso)
+	return &V8Instance{iso, ctx}, nil
+}
+
+func (inst *V8Instance) Cleanup(ctx context.Context) error {
+	return nil
+}
```

Comment:
What is cleanup doing? I mean, I understand it's coming from the interface, but doing nothing requires a comment here to explain why there is no cleanup to do.